pkg/kubelet/Dockerfile

```diff
@@ -55,17 +94,20 @@ RUN apk add --no-cache --initdb -p /out \
     util-linux \
```

Comment:
I'm pretty sure we can get rid of `libc6-compat` now, wdyt?

Reply:
I can't remember what it was needed for (maybe it was even cargo-culted from somewhere else and never useful here). If someone wants to remove it and test the result that would be great.

Reply:
@ilj it was needed to run Kubernetes release binaries, it's not needed since we build here, although I don't know about the CNI binaries, but that's not a question as of this PR. I've put it there in the first place.

Reply:
Ah, yes, it's probably useless then.

I'll try and pin a commit on here before I head off for vacation then!